bin/echo.c

```diff
@@ -138,6 +141,11 @@ int negotiate(struct s2n_connection *conn, int fd)
         printf("Resumed session\n");
     }
 
+    if (conn->psk_params.chosen_psk) {
```

Comment:
Will customers have access to this symbol?

Should you add an API to retrieve the information you need instead?